### PR TITLE
Refactor Graph enum values into class

### PR DESCRIPTION
**Pull request by inverse**
What do you think of this kind of encapsulation for "enum" type objects.

You could go more strict and use https://docs.python.org/3/library/enum.html

But I think for now this is a step in the right direction. 

If you like this approach I can update the remaining.

### CODE REVIEW

**Comment by ZENALC** [comment]
Yes, this is a TODO that has been in the backlog for a while. We really need to use real Enums for all the "enums" that we currently have. 

That way we don't need a function to map an enum to its string representation and so on and so forth.

Using an enum for graph types is a fantastic idea. While we're at it, we should probably convert traders, trends, and other things in the enums file to actual enums. This way, our imports will be a lot cleaner too as we don't need to import a lot of constants :D

**Comment by ZENALC** [comment]
I suppose we should split into several PRs as there are a LOT of constants. However, what do you think?

**Comment by inverse** [comment]
Don't mind creating smaller PR's for each enum type. Would make reviewing easier :)

File: algobot/graph_helpers.py [hunk 1]
```diff
@@ -133,24 +133,24 @@ def add_data_to_plot(gui: QMainWindow, targetGraph: PlotWidget, plotIndex: int,
...
 
-    if graphType == NET_GRAPH:
+    if graph_type == GraphType.NET:
         setup_net_graph_plot(gui, graph=graph, trader=trader, color=colors[0])
-    elif graphType == AVG_GRAPH:
+    elif graph_type == GraphType.AVG:
         setup_average_graph_plots(gui, graph=graph, trader=trader, colors=colors)
     else:
-        raise TypeError("Invalid type of graph provided.")
+        raise TypeError(f"Invalid type ({graph_type}) of graph provided.")
```
Comment (ZENALC) [inline algobot/graph_helpers.py:153]: beautiful

Comment (ZENALC) [reply algobot/graph_helpers.py:153]: Only thing is, since these are just integers, the value is not that useful, but let's resolve in another PR.

**Review by ZENALC — APPROVED** [review]

**Pull request merged**